commit 1661ec2fd8296c899fea34eeba07efeda635cc79 (refs/changes/90/5090/2)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2018-11-21T19:13:08+00:00 (11 months ago)
    
    MANTA-4024 madtom doesn't use its nodeunit dependency (fix build)
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index 9be8549..0ca1640 100644
--- a/Makefile
+++ b/Makefile
@@ -73,7 +73,7 @@ NPM_ENV		 = MAKE_OVERRIDES="CTFCONVERT=/bin/true CTFMERGE=/bin/true"
 # Repo-specific targets
 #
 .PHONY: all
-all: $(SMF_MANIFESTS) | $(REPO_DEPS) scripts
+all: $(SMF_MANIFESTS) | $(NPM_EXEC) $(REPO_DEPS) scripts
 	$(NPM) install
 
 CLEAN_FILES += $(NODEUNIT) ./node_modules/nodeunit
